### DISCUSSION TITLE
Can we safely set inner HTML?

### DISCUSSION THREAD

**Discussion by ghost** [framesurge/perseus#216] [General]
There's a variable or type "dangerously_set_inner_html" in [examples/website/state_generation/src/main.rs](https://github.com/framesurge/perseus/blob/main/examples/website/state_generation/src/main.rs) , can we safely set inner HTML?

**Comment by lukechu10** [comment]
It's actually part of sycamore and it basically allows you to inject raw html strings into the view. There is a short section about it here: https://sycamore-rs.netlify.app/docs/basics/view#dangerously_set_inner_html
